vision/cloud-client/detect/vision_async_batch_annotate_images.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Google LLC
```

Comment:
I think we want to bump to 2020 since it'll technically be a new file for the master branch.

Reply:
Done.